Plugins~/Src/MeshSyncClientBlender/BlenderPyObjects/BlenderPyID.h

```diff
@@ -13,10 +13,11 @@ class BlenderPyID
 public:
     MSBLEN_BOILERPLATE2(BlenderPyID, ID)
 
-    const char *name() const;
+        const char* name() const;
```

Comment:
something is going on with the identation here

Reply:
fixed